src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -40,6 +48,18 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
     }
 
     this._config = config;
+    this._cardSize = config._card_size;
+
+    this._disconnect();
```

Comment:
I think that we should also connect if `hass` is available. That way we can change the template and a new subscription is created when using the UI editor.

```ts
if (this._hass){ 
  this._connect();
}
```